README.md

```diff
@@ -424,6 +424,8 @@ downloadLicenses {
 }
 ```
 
+**A note on Gradle Subprojects**: `license_finder` does not report Gradle subprojects as project roots when generating reports. However, if there is a non-Gradle package definition file in a subproject, then it will be included in the report.
```

Comment:
This needs to be updated, as this change only affects 'project_roots' and not 'reports'